torchaudio/functional.py

```diff
@@ -218,42 +218,51 @@ def istft(
 def spectrogram(
     waveform, pad, window, n_fft, hop_length, win_length, power, normalized
 ):
-    # type: (Tensor, int, Tensor, int, int, int, int, bool) -> Tensor
+    # type: (Tensor, int, Tensor, int, int, int, Optional[int], bool) -> Tensor
     r"""
     spectrogram(waveform, pad, window, n_fft, hop_length, win_length, power, normalized)
 
     Create a spectrogram from a raw audio signal.
 
     Args:
-        waveform (torch.Tensor): Tensor of audio of dimension (channel, time)
+        waveform (torch.Tensor): Tensor of audio of dimension ([batch,] channel, time)
```

Comment:
It's not directly about this PR, but probably the optional last dimension for complex stft should be mentioned as well.

Reply:
The convention was specifying the optional batch dimension as `(*, channel, time)`. Maybe it's nicer to follow?

Reply:
Yes, I agree, thanks for pointing this out!